.github/workflows/release.yml

```diff
@@ -37,9 +37,9 @@ jobs:
         path: |
           ~/.cargo/registry
           ~/.cargo/git
-          target
-        key: ${{ runner.os }}-cargo-${{ github.job }}-${{ secrets.CI_CACHE_VERSION }}
-    - run: cargo test
+          target/release
+        key: release-${{ runner.os }}-${{ github.job }}-${{ secrets.CI_CACHE_VERSION }}
+    - run: cargo test --release
```

Comment:
The release job now runs tests in release mode (as it should have from the start).